packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -21,7 +21,7 @@ import MessageList from './messagelist';
 const CHAT_CLASS = 'dx-chat';
 const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 
-type Properties = ChatProperties & { title: string };
+type Properties = ChatProperties & { title: string; showDayHeaders: boolean };
```

Comment:
I hope you will remove this hack when the types are added to the d.ts file.

Reply:
I've already added note about this in .d.ts update trello card